elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -282,12 +288,18 @@ export abstract class BaseTransaction {
 		return createResponse(this.id, errors);
 	}
 
-	public apply(store: StateStore): TransactionResponse {
+	public apply(
+		networkIdentifier: string,
```

Comment:
`networkIdentifier` should be am instance property IMO.
This way you don't need to pass it as parameter for every base transaction method.